src/nimodinst/tests/test_modinst.py

```diff
@@ -1,74 +0,0 @@
-
```

Comment:
Don't delete the ModInst unit tests. I filed issue #221 because we need to re-enable running these tests.